### ISSUE TITLE
Processors for image masks

### ISSUE THREAD

**Issue by Saquib764** [comfyanonymous/ComfyUI#1160]
During my work flow, I frequently dilate/erode the masks to get the desired result. Currently, this has to be done outside ComfyUI, in python. It would be convenient to support this natively to speed up experimentation.

**Comment by h3clikejava** [comment]
Do you have the same requirement as me? Have you solved it through automation?

**Comment by h3clikejava** [comment]
I have the same requirement as you, Have you solved it through automation?

**Comment by Saquib764** [comment]
Yes, I built a custom block using Kornia library, which works with pytorch tensor data type.

https://kornia.readthedocs.io/en/latest/morphology.html

But it would be nice to have this as native feature.

**Comment by ducha-aiki** [comment]
https://github.com/comfyanonymous/ComfyUI/pull/2781 This PR doesn't work (yet) for masks, but you can convert masks to images, process, and then convert back
